pkgs/by-name/ch/choose-gui/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "choose-gui";
+  version = "1.3.1";
+
+  src = fetchurl {
```

Comment:
Consider using `fetchFromGitHub`. (That's the preferable option.)

Reply:
I think I'm going to leave it as I have it now because it looks like the build process relies on some XCode tooling. I will investigate further tomorrow, though -- would probably be better to build the binary via Nix instead of relying on pre-built binaries.

Can `fetchFromGitHub` be used to to grab assets from a release?

Reply:
It is ideal to build everything from sources.
If some binary is being used, it has to be flagged as a binary package.

Reply:
This is resolved on the latest commit I pushed :smile: